src/pointerToPath.ts

```diff
@@ -0,0 +1,37 @@
+import { replaceInString } from './_utils';
+
+export const pointerToPath = (pointer: string): string[] => {
+  return decodeUriFragmentIdentifier(pointer);
+};
+
+const decodeFragmentSegments = (segments: string[]): string[] => {
+  const len = segments.length;
+  const res = new Array(len);
```

Comment:
In this particular case, it might be better to start off with an empty array and resize it later. [See](https://v8.dev/blog/elements-kinds#performance-tips) for reference